2017/AUG/AUG17/GCAC/GCAC.cpp

```diff
@@ -0,0 +1,48 @@
+//Made by vanguard7
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios::sync_with_stdio(false);
+	cin.tie(NULL);
+	int t;
+	cin >> t;
+	while(t--) {
+		int n, m, i, j;
+		set <int> s;
+		cin >> n >> m;
+		char x[m+1];
+		long long int minSalary[n], offeredSalary[m], maxJobOffered[m], cost = 0, jobs = 0, companies = m;
+		for(i = 0; i < n; ++i)
```

Comment:
Add the statements in a block even if it is one line, enhances code readability when the codes are large.

Reply:
I think this is not needed, as we are making code readable here, not standard.

Reply:
I'm okay with either 😄 .

Reply:
@Monal5031 Kindly approve.